src/utils/slpFileWriter.ts

```diff
@@ -8,18 +6,20 @@ import { SlpFile } from "./slpFile";
 import type { SlpRawEventPayload, SlpStreamSettings } from "./slpStream";
 import { SlpStream, SlpStreamEvent } from "./slpStream";
 
+import { format } from "date-fns";
+
 /**
  * The default function to use for generating new SLP files.
  */
-function getNewFilePath(folder: string, m: Moment): string {
-  return path.join(folder, `Game_${m.format("YYYYMMDD")}T${m.format("HHmmss")}.slp`);
+function getNewFilePath(folder: string, d: Date): string {
```

Comment:
```suggestion
function getNewFilePath(folder: string, date: Date): string {
```
I know you just copied it from the old function but we should probably give this a proper variable name.

Reply:
Yeah that probably makes sense, I'll finish refactoring on my machine